### ISSUE TITLE
Loosing position in lighttable when the actual picture is rejected

### ISSUE THREAD

**Issue by wwwFrank** [darktable-org/darktable#17413]
### Describe the bug

After import the result of a day of shooting into Darktable the next step is a quality-control to reduce the number of pictures. From similar pictures choose the best and reject the rest.
I set Filter: "selected all exept rejected", then walk through the folder, look deeper in a picture with "w"-key, give a star-rating (key 0 - 5) or reject with "r"-key. Rejected pictures are filtered out to delete them later.
After rejecting a picture from the lighttable the view jumps to the next picture, which is OK.
To have a detailed and deep look into a picture (realy sharp?) I have to doubleclick, go into the darkroom and zoom in to a full resolution (size 100%).
When I reject the picture in the darkroom (key "r") an then return to lighttable (key "l") then the position is lost and the view jumps back to the first picture of the folder. I have to scroll again through the list to find the pictures which have not been checked.

### Steps to reproduce

1. Set Filter: "selected all exept rejected"
2. select a picture in the middle of the list
3. key d (darkroom)
4. key r (reject) in darkroom
5. Key l (lightable)
6. The view jumps to the first picture

### Expected behavior

Please go to the next picture when returning from darkroom

### Where did you obtain darktable from?

OBS

### darktable version

4.8.1

### What OS are you using?

Linux

### What is the version of your OS?

Ubuntu 22.4

### Are you using OpenCL GPU in darktable?

Yes

**Comment by wpferguson** [comment]
You can use full screen cullilng in lighttable.  When you want to zoom in to 100% then just middle click.  Middle click again to zoom out.  You may also be able to use sticky preview and middle click to get 100%.

**Comment by jsmucr** [comment]
Yep, that's annoying. And more so because you don't even need to leave the darkroom to lose the position when removing more than one image.

**Comment by wwwFrank** [comment]
> When you want to zoom in to 100% then just middle click

"middle click" in lighttable don't work for me, but that's not it. Let let me say it more generally:
While working in the darkroom you change the star-rating or the color-label and then return to lighttable.
The filter no longer contains the modified image. The position is lost, you drop back to image #1 of the collection.

**Comment by wwwFrank** [comment]
> "middle click" in lighttable don't work for me

Just got it! Middle click while holding key "w" is working.
